sklearn/preprocessing/tests/test_data.py

```diff
@@ -1588,3 +1590,13 @@ def test_fit_cold_start():
         # with a different shape, this may break the scaler unless the internal
         # state is reset
         scaler.fit_transform(X_2d)
+
+
+def test_boxcox_transform():
```

Comment:
Can you put in a comment what is the working principle of this test: how do we know that it's correct.